public/data/nikhil.json

```
@@ -0,0 +1,19 @@
+{
+    "name": "nikhil",
+    "type": "personal",
+    "bio": "aspiring software engineer",
+    "avatar": "https://avatars.githubusercontent.com/u/72465503?s=400&u=d87f27cebe9055972284614285014b72bbcec3b0&v=4",
```

I recommend you to use this url, as it is more dynamic in case you change your username!!
```suggestion
    "avatar": "https://github.com/nikhilxe.png",
```